.github/workflows/periodic.yml

```diff
@@ -0,0 +1,33 @@
+name: Periodic Functional Tests
+
+on:
+  schedule:
+    - cron: '*/5 * * * *' # Run every 5 minutes
+
+jobs:
+  run_tests:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      jobs:
+      setup_kind:
+      runs-on: ubuntu-latest
+      steps:
+    - name: Create k8s Kind Cluster
+      uses: container-tools/kind-action@v2
+      with:
+        cluster_name: kfp-tekton
+        kubectl_version: v1.29.2
+        version: v0.22.0
+        node_image: kindest/node:v1.29.2      
+    - name: Run Functional Tests
+      run: 
+        ./test/kfp-functional-test/kfp-functional-test.sh > periodic_tests.txt
```

Comment:
Let me suggest a different approach:
```
log_dir=$(mktemp -d)
./test/kfp-functional-test/kfp-functional-test.sh > $log_dir/periodic_tests.txt
```

Reply:
Why `mktemp`? It creates a random path under `/tmp` for your temporary artifact collection task. That way, the last step only needs to get all the paths created by `mktemp`, compress in a tarball, and publish to a remote path.

Reply:
I've made the changes you suggested and included the artifact collection, @rimolive.